### PR TITLE
Update ecosystem-tool-user-case.md

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

更新内容为表格。后续可能会和其他内容合并。

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ecosystem-tool-user-case.md [hunk 1]
```diff
@@ -6,34 +6,13 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
+阅读本文档，你可以了解在特定使用场景中需要使用哪些工具。
+
+| 使用场景 |迁移源|迁移目的地| 工具 | 描述 | 
+|:---|:---|:---|:---|:---|
+|CSV 数据导入  |CSV|TiDB| [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md) |当需要将其他工具导出的格式兼容的 CSV files 导入到 TiDB 时，可使用 [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md)。 |  
+| 全量数据迁移  | MySQL/Aurora |TiDB|  [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump files，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。 | 
+| 全量 + 增量 数据迁移 | MySQL/Aurora |TiDB|[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview)  | 当既需要从 MySQL/Aurora 导入全量数据，又需要迁移增量数据时，可使用 [TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview) 完成[全量数据和增量数据的迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)。 如果全量数据量较大（TB 级别），则可先使用 [Dumpling](/dumpling-overview.md) 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 完成全量数据的迁移，再使用 DM 完成增量数据的迁移。|
```
Comment (Liuxiaozhen12) [inline ecosystem-tool-user-case.md:15]:
```suggestion
| 全量 + 增量 数据迁移 | MySQL/Aurora |TiDB|[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview)  | 当你既需要从 MySQL/Aurora 导入全量数据，又需要迁移增量数据时，可使用 [TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview) 完成[全量数据和增量数据的迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)。 如果全量数据量较大（TB 级别），则可先使用 [Dumpling](/dumpling-overview.md) 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 完成全量数据的迁移，再使用 DM 完成增量数据的迁移。| 

```

File: ecosystem-tool-user-case.md [hunk 2]
```diff
@@ -6,34 +6,13 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
+| 全量数据迁移  | MySQL/Aurora |TiDB|  [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump files，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。 | 
+| 全量 + 增量 数据迁移 | MySQL/Aurora |TiDB|[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview)  | 当既需要从 MySQL/Aurora 导入全量数据，又需要迁移增量数据时，可使用 [TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview) 完成[全量数据和增量数据的迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)。 如果全量数据量较大（TB 级别），则可先使用 [Dumpling](/dumpling-overview.md) 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 完成全量数据的迁移，再使用 DM 完成增量数据的迁移。| 
+| TiDB 集群备份与恢复 |TiDB |TiDB|[BR](/br/backup-and-restore-tool.md)  | 当需要对 TiDB 集群进行备份或在之后对 TiDB 集群进行恢复时，可使用 [BR](/br/backup-and-restore-tool.md)。另外，BR 也可以对 TiDB 的数据进行[增量备份](/br/use-br-command-line-tool.md#增量备份)和[增量恢复](/br/use-br-command-line-tool.md#增量恢复)。 |
```
Comment (Liuxiaozhen12) [inline ecosystem-tool-user-case.md:16]: 不太懂第一句的 `在之后`是什么意思，在什么之后呢？

File: ecosystem-tool-user-case.md [hunk 3]
```diff
@@ -6,34 +6,13 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
+| 全量 + 增量 数据迁移 | MySQL/Aurora |TiDB|[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview)  | 当既需要从 MySQL/Aurora 导入全量数据，又需要迁移增量数据时，可使用 [TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview) 完成[全量数据和增量数据的迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)。 如果全量数据量较大（TB 级别），则可先使用 [Dumpling](/dumpling-overview.md) 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 完成全量数据的迁移，再使用 DM 完成增量数据的迁移。| 
+| TiDB 集群备份与恢复 |TiDB |TiDB|[BR](/br/backup-and-restore-tool.md)  | 当需要对 TiDB 集群进行备份或在之后对 TiDB 集群进行恢复时，可使用 [BR](/br/backup-and-restore-tool.md)。另外，BR 也可以对 TiDB 的数据进行[增量备份](/br/use-br-command-line-tool.md#增量备份)和[增量恢复](/br/use-br-command-line-tool.md#增量恢复)。 | 
+| TiDB 之间全量 + 增量 数据迁移|TiDB |TiDB| [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要将 TiDB 集群的数据迁出到其他 TiDB 集群时，可使用 [Dumpling](/dumpling-overview.md) 从 TiDB 将全量数据导出为 SQL dump files，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB。如果还需要执行增量数据的迁移，则可使用 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。 |
```
Comment (Liuxiaozhen12) [inline ecosystem-tool-user-case.md:17]:
```suggestion
| TiDB 之间全量 + 增量 数据迁移|TiDB |TiDB| [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要将 TiDB 集群的数据迁移到其他 TiDB 集群时，可使用 [Dumpling](/dumpling-overview.md) 从 TiDB 将全量数据导出为 SQL dump 文件，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB。如果还需要执行增量数据的迁移，可使用 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。 | 

```

**Review by Liuxiaozhen12 — APPROVED** [review]
rest LGTM

File: ecosystem-tool-user-case.md (hunk 2 above)
Comment (hfxsd) [reply ecosystem-tool-user-case.md:20]: 应该是备份之后，再恢复数据。

File: ecosystem-tool-user-case.md [hunk 4]
```diff
@@ -6,34 +6,18 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
 # TiDB 生态工具适用场景
 
-本文档从生态工具的适用场景出发，介绍部分常见场景下的生态工具选择。
+阅读本文档，你可以了解在特定使用场景中需要使用哪些工具。
 
-## 从 CSV 导入数据到 TiDB
+| 使用场景 |迁移源|迁移目的地| 工具 | 描述 | 
+|:---|:---|:---|:---|:---|
+|CSV 数据导入  |CSV|TiDB| [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md) |当需要将其他工具导出的格式兼容的 CSV 文件导入到 TiDB 时，可使用 [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md)。 |  
+`格式兼容`是指 CSV 文件与 TiDB 格式兼容对吗？
```
Comment (TomShawn) [inline ecosystem-tool-user-case.md:15]:
```suggestion
```

File: ecosystem-tool-user-case.md [hunk 5]
```diff
@@ -6,34 +6,18 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
 
-当需要将其他工具导出的格式兼容的 CSV files 导入到 TiDB 时，可使用 [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md)。
+| 全量数据迁移  | MySQL/Aurora |TiDB|  [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump 文件，再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。 |
```
Comment (TomShawn) [inline ecosystem-tool-user-case.md:17]:
```suggestion
```

File: ecosystem-tool-user-case.md [hunk 6]
```diff
@@ -6,34 +6,18 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
+`格式兼容`是指 CSV 文件与 TiDB 格式兼容对吗？
 
-当需要将其他工具导出的格式兼容的 CSV files 导入到 TiDB 时，可使用 [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md)。
+| 全量数据迁移  | MySQL/Aurora |TiDB|  [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump 文件，再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。 | 
 
-## 从 MySQL/Aurora 导入全量数据
+| 全量 + 增量 数据迁移 | MySQL/Aurora |TiDB|[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview)  | 当你既需要从 MySQL/Aurora 导入全量数据，又需要迁移增量数据时，可使用 [TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview) 完成[全量数据和增量数据的迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)。 如果全量数据量较大（TB 级别），则可先使用 [Dumpling](/dumpling-overview.md) 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 完成全量数据的迁移，再使用 DM 完成增量数据的迁移。|
```
Comment (TomShawn) [inline ecosystem-tool-user-case.md:19]:
```suggestion
```

File: ecosystem-tool-user-case.md [hunk 7]
```diff
@@ -6,34 +6,18 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
 
-当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump files，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。
+| TiDB 集群备份与恢复 |TiDB |TiDB|[BR](/br/backup-and-restore-tool.md)  | 当需要对 TiDB 集群进行备份或在之后对 TiDB 集群进行恢复时，可使用 [BR](/br/backup-and-restore-tool.md)。另外，BR 也可以对 TiDB 的数据进行[增量备份](/br/use-br-command-line-tool.md#增量备份)和[增量恢复](/br/use-br-command-line-tool.md#增量恢复)。 | 
+| TiDB 之间全量 + 增量 数据迁移|TiDB |TiDB| [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要将 TiDB 集群的数据迁移到其他 TiDB 集群时，可使用 [Dumpling](/dumpling-overview.md) 从 TiDB 将全量数据导出为 SQL dump 文件，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB。如果还需要执行增量数据的迁移，可使用 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。 |
```
Comment (TomShawn) [inline ecosystem-tool-user-case.md:22]:
```suggestion
```

File: ecosystem-tool-user-case.md [hunk 8]
```diff
@@ -6,34 +6,18 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
 # TiDB 生态工具适用场景
 
-本文档从生态工具的适用场景出发，介绍部分常见场景下的生态工具选择。
+阅读本文档，你可以了解在特定使用场景中需要使用哪些工具。
 
-## 从 CSV 导入数据到 TiDB
+| 使用场景 |迁移源|迁移目的地| 工具 | 描述 | 
+|:---|:---|:---|:---|:---|
+|CSV 数据导入  |CSV|TiDB| [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md) |当需要将其他工具导出的格式兼容的 CSV 文件导入到 TiDB 时，可使用 [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md)。 |  
+`格式兼容`是指 CSV 文件与 TiDB 格式兼容对吗？
```
Comment (TomShawn) [inline ecosystem-tool-user-case.md:14]:
```suggestion
```

File: ecosystem-tool-user-case.md [hunk 9]
```diff
@@ -6,34 +6,18 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
 
-当需要将其他工具导出的格式兼容的 CSV files 导入到 TiDB 时，可使用 [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md)。
+| 全量数据迁移  | MySQL/Aurora |TiDB|  [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump 文件，再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。 |
```
Comment (TomShawn) [inline ecosystem-tool-user-case.md:16]:
```suggestion
| 全量数据迁移  | MySQL/Aurora |TiDB|  [Dumpling](/dumpling-overview.md)，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump 文件，再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。 | 

```

File: ecosystem-tool-user-case.md [hunk 10]
```diff
@@ -6,34 +6,18 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
+`格式兼容`是指 CSV 文件与 TiDB 格式兼容对吗？
 
-当需要将其他工具导出的格式兼容的 CSV files 导入到 TiDB 时，可使用 [TiDB Lightning](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md)。
+| 全量数据迁移  | MySQL/Aurora |TiDB|  [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump 文件，再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。 | 
 
-## 从 MySQL/Aurora 导入全量数据
+| 全量 + 增量 数据迁移 | MySQL/Aurora |TiDB|[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview)  | 当你既需要从 MySQL/Aurora 导入全量数据，又需要迁移增量数据时，可使用 [TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview) 完成[全量数据和增量数据的迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)。 如果全量数据量较大（TB 级别），则可先使用 [Dumpling](/dumpling-overview.md) 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 完成全量数据的迁移，再使用 DM 完成增量数据的迁移。|
```
Comment (TomShawn) [inline ecosystem-tool-user-case.md:18]:
```suggestion
| 全量 + 增量 数据迁移 | MySQL/Aurora |TiDB|[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview)  | 当你既需要从 MySQL/Aurora 导入全量数据，又需要迁移增量数据时，可使用 [TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview) 完成[全量数据和增量数据的迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)。 如果全量数据量较大（TB 级别），则可先使用 [Dumpling](/dumpling-overview.md) 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 完成全量数据的迁移，再使用 DM 完成增量数据的迁移。| 
```

File: ecosystem-tool-user-case.md [hunk 11]
```diff
@@ -6,34 +6,18 @@ aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
...
 
-当需要从 MySQL/Aurora 导入全量数据时，可先使用 [Dumpling](/dumpling-overview.md) 将数据导出为 SQL dump files，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB 集群。
+| TiDB 集群备份与恢复 |TiDB |TiDB|[BR](/br/backup-and-restore-tool.md)  | 当需要对 TiDB 集群进行备份或在之后对 TiDB 集群进行恢复时，可使用 [BR](/br/backup-and-restore-tool.md)。另外，BR 也可以对 TiDB 的数据进行[增量备份](/br/use-br-command-line-tool.md#增量备份)和[增量恢复](/br/use-br-command-line-tool.md#增量恢复)。 | 
+| TiDB 之间全量 + 增量 数据迁移|TiDB |TiDB| [Dumpling](/dumpling-overview.md) ，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要将 TiDB 集群的数据迁移到其他 TiDB 集群时，可使用 [Dumpling](/dumpling-overview.md) 从 TiDB 将全量数据导出为 SQL dump 文件，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB。如果还需要执行增量数据的迁移，可使用 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。 |
```
Comment (TomShawn) [inline ecosystem-tool-user-case.md:21]:
```suggestion
| TiDB 之间全量 + 增量 数据迁移|TiDB |TiDB| [Dumpling](/dumpling-overview.md)，[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) | 当需要将 TiDB 集群的数据迁移到其他 TiDB 集群时，可使用 [Dumpling](/dumpling-overview.md) 从 TiDB 将全量数据导出为 SQL dump 文件，然后再使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 将数据导入到 TiDB。如果还需要执行增量数据的迁移，可使用 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)。 | 

```

File: ecosystem-tool-user-case.md (hunk 10 above)
Comment (TomShawn) [reply ecosystem-tool-user-case.md:18]:
```suggestion
| 全量 + 增量数据迁移 | MySQL/Aurora |TiDB|[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview)  | 当你既需要从 MySQL/Aurora 导入全量数据，又需要迁移增量数据时，可使用 [TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/overview) 完成[全量数据和增量数据的迁移](https://docs.pingcap.com/zh/tidb-data-migration/v2.0/migrate-from-mysql-aurora)。 如果全量数据量较大（TB 级别），则可先使用 [Dumpling](/dumpling-overview.md) 与 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 完成全量数据的迁移，再使用 DM 完成增量数据的迁移。| 

```

**Comment by TomShawn** [comment]
见预览：https://github.com/pingcap/docs-cn/blob/73e080461dbff8d86f3041cfc7859fc276bdff1e/ecosystem-tool-user-case.md
表格仍是破碎的。建议每个分隔符 `|` 与字符之间至少保留一个空格。

**Pull request closed without merging**